documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1430,11 +1430,11 @@ Note that primary key columns are always included in the event's key, also if bl
 Based on the used hash function referential integrity is kept while data is pseudonymized. Supported hash functions are described in the {link-java7-standard-names}[MessageDigest section] of the Java Cryptography Architecture Standard Algorithm Name Documentation.
 The hash is automatically shortened to the length of the column.
 
-Multiple properties with different lengths can be used in a single configuration, although in each the length must be a positive integer or zero. Fully-qualified names for columns are of the form _databaseName_._tableName_._columnName_, or _databaseName_._schemaName_._tableName_._columnName_.
+Multiple properties with different lengths can be used in a single configuration, although in each the length must be a positive integer or zero. Fully-qualified names for columns are of the form _databaseName_._schemaName_._tableName_._columnName_.
 
 Example:
 
-    column.mask.hash.SHA-256.with.salt.CzQMA0cB5K = inventory.orders.customerName, inventory.shipment.customerName
+    column.mask.hash.SHA-256.with.salt.CzQMA0cB5K = testDB.dbo.orders.customerName, testDB.dbo.shipment.customerName
```

Comment:
So can I indeed capture changes from multiple databases with a single connector instance? // CC @jpechane

Reply:
No we can't.  In SQL Server we specifically use the `database.dbname` configuration option to drive which database the driver connects to and all subsequent queries we use to fetch object information is based on the name returned from `select db_name()`.  Therefore, its not possible for the connector to monitor different objects from varying databases even if they exist in the same database server instance from what I can see.

When we fetch the list of change tables, we build the `TableId` and use the database name as the catalog portion of the identifier.  So for matching purposes, the column.mask.xxx options would need to either use a regex for explicit string matches that follow the form of `<catalog>.<schema>.<table>.<column>`.

Reply:
> Therefore, its not possible for the connector to monitor different objects from varying databases

Ok, that's what I thought.

> would need to either use a regex for explicit string matches that follow the form of <catalog>.<schema>.<table>.<column>

But what's the purpose/benefit of adding the catalog part, if it's constant and the same for all cases when looking at a single connector? We also don't require the database name in case of Postgres (which is a comparable situation).